ToggleLockBranch/ReadMe.md

```diff
@@ -40,7 +40,9 @@ None
 
 ## Prerequisites
 
-Send email to REs - ask to add a `github_lock` token to the Vault configuration of your repository - for each repository where you need to use this. [Example](https://github.com/SonarSource/re-terraform-aws-vault/blob/3f5efb03a94a38d1346cbde62631ce1a340b5e14/orders/bubble-dotnet.yaml#L180-L182)
+Ask DevInfra squad to add a 
+* `github_lock` token to the Vault configuration of your repository. [Example](https://github.com/SonarSource/re-terraform-aws-vault/blob/3f5efb03a94a38d1346cbde62631ce1a340b5e14/orders/bubble-dotnet.yaml#L180-L182)
+* `development/kv/data/slack` token. [Example](https://github.com/SonarSource/re-terraform-aws-vault/blob/3f5efb03a94a38d1346cbde62631ce1a340b5e14/orders/bubble-dotnet.yaml#L186)
```

Comment:
I think this can be achieved through SPEED with the `Manage Vault Policy` action.
More info on the [xtranet](https://xtranet-sonarsource.atlassian.net/wiki/spaces/Platform/pages/3309240583/End-user+Documentation+-+SPEED#Catalog---Vault-Permissions).

Technically it is still _asking DevInfra_.

Reply:
It can not :) Speed can give only trivial tickets.